pkg/config/sink.go

```diff
@@ -17,6 +17,7 @@ package config
 type SinkConfig struct {
 	DispatchRules []*DispatchRule `toml:"dispatchers" json:"dispatchers"`
 	Protocol      string          `toml:"protocol" json:"protocol"`
+	OldValue      bool            `toml:"old-value" json:"old-value"`
```

Comment:
Shall we add `old-value` config to `cmd/changefeed.toml` now, and add comment to address this is an **experimental feature**.